data/young2020epidemiologic/young2020epidemiologic.yaml

```diff
@@ -0,0 +1,434 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2
+  in Singapore
+doi: 10.1001/jama.2020.3204
+description: >
+  Clinical, laboratory, and radiologic data were collected, including PCR cycle threshold
+  values from nasopharyngeal swabs and viral shedding in blood, urine, and stool.
+  Clinical course was summarized, including requirement for supplemental oxygen and
+  intensive care and use of empirical treatment with lopinavir-ritonavir. The numbers
+  of positive stool, blood, and urine samples were small. The data was obtained from
+  Goyal et al. 2020 for the nasopharyngeal swab results in 11 patients.
+analytes:
+  naso_swab_SARSCoV2:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration in nasopharyngeal swab samples. The
+      concentration were quantified in genome copies per swab.
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N, E, and ORF1lab
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    unit: gc/swab
+    reference_event: confirmation date
+participants:
+- measurements:
+  - analyte: naso_swab_SARSCoV2
+    time: 0
+    value: 4.29246
+  - analyte: naso_swab_SARSCoV2
+    time: 6
+    value: 2.74158
+  - analyte: naso_swab_SARSCoV2
+    time: 8
+    value: 3.51702
+  - analyte: naso_swab_SARSCoV2
+    time: 10
+    value: negative
+  - analyte: naso_swab_SARSCoV2
+    time: 11
+    value: negative
+  - analyte: naso_swab_SARSCoV2
+    time: 12
+    value: 3.1293
```

Comment:
Should these be raised to the power of ten, e.g., `10 ** 3.1293 = 1346.7903626291`?

Reply:
Just fixed them! Thank you.